README.md

```diff
@@ -91,55 +92,50 @@ This will copy the GitHub repository contents into a new folder in the current d
 cd mm
 ```
 
-#### 3. Install python dependencies
+#### 3. Prepare a base ROM
 
-The build process has a few python packages required that are located in `requirements.txt`.
+Place a copy of the US ROM inside the `baseroms/n64-us/` folder.
 
-It is recommend to setup a virtual environment for python to localize all dependencies. To create a virtual environment:
+Rename the file to `baserom.z64`, `baserom.n64` or `baserom.v64`, depending on the original extension.
 
-```bash
-python3 -m venv .mm-env
-```
-
-To activate or deactivate the virtual environment run either:
-```bash
-# Activates the mm-env virtual environment
-source .mm-env/bin/activate 
-# Deactivates the active virtual environment
-deactivate
-```
-
-Once activated you can install the required dependencies:
-```bash
-pip install -r requirements.txt  # or python3 -m pip if you want
-```
-
-**Important:** This virtual environment will need to be activated everytime you enter the repo.
-
-#### 4. Prepare a base ROM
-
-Copy your ROM to inside the root of this new project directory, and rename the file of the baserom to reflect the version of ROM you are using. ex: `baserom.mm.us.rev1.z64`
-
-#### 5. Make and Build the ROM
+#### 4. Make and Build the ROM
 
 To start the extraction/build process, run the following command:
 
 ```bash
 make init
 ```
 
-This will extract all the individual files in the ROM into a newly created baserom folder, as well as decompress the compressed files in a newly created decomp folder. This will create the build folders as well as a new folder with the ASM as well as containing the disassemblies of nearly all the files containing code.
+The extraction/build process:
+1. Fetches dependencies: Downloads necessary tools from pip.
+2. Prepares build environment: Compiles tools for the build process.
+3. Extracts ROM contents:
+    - Creates a baserom folder to house individual files extracted from the ROM.
+    - Decompresses compressed files into a decomp folder.
+4. Disassembles code:
+    - Creates folders for the build process.
+    - Generates a folder containing ASM files and disassemblies of most code-containing files.
+5. Builds the ROM
```

Comment:
This is missing asset extraction, isn't it?

Reply:
Expanded on this. Let me know if I should add/change anything.